public/index.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+$filename = __DIR__.preg_replace('#(\?.*)$#', '', isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '');
```

Comment:
I don't really like adding code that's used solely for development purposes, especially when things like Valet and Homestead exist. Did you try with Valet/Homestead and were having trouble?